Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,20 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+// const penceString = "399p";
+
+function changeToPound(input) {
```

Comment:
the current it's fine, just a suggestion another good one could be `convertPennysToPounds`

Reply:
works like a charm  :)
<img width="390" alt="image" src="https://github.com/user-attachments/assets/a97ef158-55a6-4d17-b312-ac6d0def7b38">